git/Contributing.md

```diff
@@ -0,0 +1,58 @@
+# Contributing
+
+In this file is explained best practice for contributing on a Git version control repository hosting service such as _Github_, _Gitlab_, _Bitbucket_, etc.
+Also, there's explained flow for developing application.
+
+In flow given in post named [Successful Git Branching Model](http://nvie.com/posts/a-successful-git-branching-model/) there's detailed explanation of proposed flow, but we're trying to keep it short.
```

Comment:
Use determiner `the` for words _flow_ and _post_ since it is exactly known on which flow and on which post you refer.